src/main/java/uk/gov/companieshouse/confirmationstatementapi/eligibility/impl/CompanyOfficerValidation.java

```diff
@@ -28,20 +29,22 @@ public CompanyOfficerValidation(OfficerService officerService){
     public void validate(CompanyProfileApi companyProfileApi) throws EligibilityException, ServiceException {
         LOGGER.info("Validating Company Officers for: {}", companyProfileApi.getCompanyNumber());
         var officers = officerService.getOfficers(companyProfileApi.getCompanyNumber());
-        var officerCount = getOfficerCount(officers.getItems(), officers.getActiveCount());
+        var officerCount = getOfficerCount(officers.getItems());
         if (officerCount != null && officerCount > 1) {
             LOGGER.info("Company Officers validation failed for: {}", companyProfileApi.getCompanyNumber());
             throw new EligibilityException(EligibilityStatusCode.INVALID_COMPANY_APPOINTMENTS_MORE_THAN_ONE_OFFICER);
         }
         LOGGER.info("Company Officers validation passed for: {}", companyProfileApi.getCompanyNumber());
     }
 
-    public Long getOfficerCount(List<CompanyOfficerApi> officers, Long activeCount) {
+    public Long getOfficerCount(List<CompanyOfficerApi> officers) {
+        Long officerCount = 0L;
         for(CompanyOfficerApi i: officers) {
```

Comment:
Appreciate it wasn't added in this pr.
But could we name the variable inside the for loop to "officer" instead of "i"?